client/src/test/scala/org/http4s/client/middleware/FollowRedirectSpec.scala

```diff
@@ -21,26 +20,21 @@ class FollowRedirectSpec extends Http4sSpec {
   }
 
 
-  val defaultClient = new MockClient(route)
+  val defaultClient = MockClient(route)
   val client = FollowRedirect(1)(defaultClient)
+  val fetchBody = client.toService(_.as[String])
   
   "FollowRedirect" should {
     "Honor redirect" in {
-      val resp = client(getUri(s"http://localhost/redirect")).run
-      resp.status must_== Status.Ok
+      fetchBody =<< GET(uri("http://localhost/redirect")) must returnValue("hello")
```

Comment:
Sadly, it doesn't have a verbal alias.  I like the form, I hate the name.  Maybe we can contribute one.  `runF`?